.vscode-template/settings.json

```diff
@@ -0,0 +1,21 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+  "editor.tabSize": 2,
+  "editor.insertSpaces": true,
+
+  "files.trimTrailingWhitespace": true,
+
+  "search.exclude": {
+    "**/*.class": true,
+    "**/*.hasTasty": true,
+    "**/target/**": true,
+    "scala2-library/{doc,docs,lib,META-INF,scripts,spec,test,tools}/*": true, // only allow scala-backend/src
```

Comment:
Can you end this regexp with `/` instead of `/*` ? Same for various other regexps below.